types/audio-type/package.json

```
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/audio-type",
+    "version": "2.2.9999",
```

```suggestion
    "version": "2.2.9999",
    "type": "module",
```

must match the implementation package: https://unpkg.com/browse/audio-type@2.2.1/package.json